reference/checkouts.sf.yml

```diff
@@ -1102,7 +1102,7 @@ components:
           description: the coupon code
         displayName:
           type: string
-          description: The coupon title based on different types provided in control panel section.
+          description: The coupon name displayed on the storefront.
```

Comment:
changed to be as same as in storefront cart api (carts.sf.yml), to be more consistent on description.

Reply:
Thanks - I wonder if we should just use the same as V3 `Display name of the coupon.` 🤷‍♂️